frontend/packages/integration-tests-cypress/views/details-page.ts

```diff
@@ -13,6 +13,7 @@ export const detailsPage = {
       .click();
   },
   isLoaded: () => cy.byTestID('skeleton-detail-view').should('not.exist'),
+  breadcrumb: (number: number) => cy.byLegacyTestID(`breadcrumb-link-${number}`),
```

Comment:
I'd probably call this `breadcrumbIndex` or similar so it's clear that's it's taking a number.

Reply:
fixed